wurst/data structures/LinkedList.wurst

```diff
@@ -163,6 +167,12 @@ public class LLIterator<Q>
 		current = current.next
 		return current.elem
 
+
+	function prev() returns Q
```

Comment:
Can you remove prev from this pull request? I think prev in combination with remove works different from the java iterators, so I would like to make this change with more time to think. Otherwise this can be merged.